python/paddle/fluid/tests/unittests/test_nextafter_op.py

```diff
@@ -0,0 +1,118 @@
+#   Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+from eager_op_test import OpTest
+
+import paddle
+
+
+def ref_nextafter(x, y):
+    out = np.nextafter(x, y)
+    return out
+
+
+class TestNextafterAPI(unittest.TestCase):
+    def setUp(self):
+        self.x = np.random.rand(1, 2).astype('float32')
+        self.y = np.random.rand(1, 2).astype('float32')
+        self.x1 = np.array([0, 0, 10]).astype("float32")
```

Comment:
why use 0 in shape? `self.x1` does not contain any data, what is the intention of this test case.

Reply:
I see pytorch nextafter is using this, so I used it,too.